src/DssSpell.sol

```diff
@@ -22,130 +22,392 @@ import "dss-exec-lib/DssAction.sol";
-
-    address public immutable MCD_VEST_DAI = DssExecLib.getChangelogAddress("MCD_VEST_DAI");
-    address public immutable MCD_VEST_MKR_TREASURY = DssExecLib.getChangelogAddress("MCD_VEST_MKR_TREASURY");
+        "2022-02-04 MakerDAO Executive Spell | Hash: TODO";
 
+    // Many of the settings that change weekly rely on the rate accumulator
+    // described at https://docs.makerdao.com/smart-contract-modules/rates-module
+    // To check this yourself, use the following rate calculation (example 8%):
+    //
+    // $ bc -l <<< 'scale=27; e( l(1.08)/(60 * 60 * 24 * 365) )'
+    //
+    // A table of rates can be found at
+    //    https://ipfs.io/ipfs/QmefQMseb3AiTapiAKKexdKHig8wroKuZbmLtPLv4u2YwW
+    //
+    uint256 constant ZERO_PCT_RATE                = 1000000000000000000000000000;
+    uint256 constant ZERO_PT_TWO_FIVE_PCT_RATE    = 1000000000079175551708715274;
+    uint256 constant ZERO_PT_SEVEN_FIVE_PCT_RATE  = 1000000000236936036262880196;
+    uint256 constant ONE_PCT_RATE                 = 1000000000315522921573372069;
+    uint256 constant ONE_PT_FIVE_PCT_RATE         = 1000000000472114805215157978;
+    uint256 constant TWO_PCT_RATE                 = 1000000000627937192491029810;
+    uint256 constant TWO_PT_TWO_FIVE_PCT_RATE     = 1000000000705562181084137268;
+    uint256 constant TWO_PT_FIVE_PCT_RATE         = 1000000000782997609082909351;
+    uint256 constant THREE_PT_SEVEN_FIVE_PCT_RATE = 1000000001167363430498603315;
+    uint256 constant FOUR_PCT_RATE                = 1000000001243680656318820312;
+    uint256 constant FIVE_PCT_RATE                = 1000000001547125957863212448;
 
-    address constant SNE_001_WALLET        = 0x6D348f18c88D45243705D4fdEeB6538c6a9191F1;
-    address constant TECH_001_WALLET       = 0x2dC0420A736D1F40893B9481D8968E4D7424bC0B;
-    address constant ORA_001_GAS           = 0x2B6180b413511ce6e3DA967Ec503b2Cc19B78Db6;
-    address constant ORA_001_GAS_EMERGENCY = 0x1A5B692029b157df517b7d21a32c8490b8692b0f;
-    address constant DUX_001_WALLET        = 0x5A994D8428CCEbCC153863CCdA9D2Be6352f89ad;
-    address constant SES_001_WALLET        = 0x87AcDD9208f73bFc9207e1f6F0fDE906bcA95cc6;
-    address constant SF_001_WALLET         = 0xf737C76D2B358619f7ef696cf3F94548fEcec379;
-    address constant RWF_001_WALLET        = 0x96d7b01Cc25B141520C717fa369844d34FF116ec;
-    address constant SF_001_VEST_01        = 0xBC7fd5AA2016C3e2C8F0dBf4e919485C6BBb59e2;
-    address constant SF_001_VEST_02        = 0xCC81578d163A04ea8d2EaE6904d0C8E61A84E1Bb;
-
-
-    uint256 constant APR_01_2021 = 1617235200;
-    uint256 constant SEP_01_2021 = 1630454400;
-    uint256 constant FEB_01_2022 = 1643673600;
-    uint256 constant DEC_31_2022 = 1672444800;
-    uint256 constant JAN_31_2023 = 1675123200;
-    uint256 constant JUL_31_2023 = 1690761600;
+    address constant NEW_MCD_ESM = 0x09e05fF6142F2f9de8B6B65855A1d56B6cfE4c58;
+    bytes32 constant MCD_ESM = "MCD_ESM";
```

Comment:
I feel it is a bit confusing that the variable naming that in general represents the address of the ESM is now used to save the string of the name.

Reply:
variable name eventually changed to MCD_ESM_NAME.